packages/core/src/hooks/useLocalStorage.ts

```diff
@@ -1,11 +1,12 @@
 import { useEffect, useState } from 'react'
+import { useConfig } from './useConfig'
 
-function getItem(key: string) {
+function getItem(key: string, localStorage: WindowLocalStorage['localStorage']) {
   if (typeof window === 'undefined') {
```

Comment:
I don't think this is necessary anymore

Reply:
I think that it is, because we create own functions from localStorage implementation.